pom.xml

```diff
@@ -47,7 +47,7 @@
     <dependency>
       <groupId>org.fcrepo</groupId>
       <artifactId>fcrepo-storage-ocfl</artifactId>
-      <version>${fcrepo-storage-ocfl.version}</version>
+      <version>6.3.0-SNAPSHOT</version>
```

Comment:
there's still a variable for `fcrepo-storage-ocfl.version` which points to 6.2.0, should we change this to updating that variable instead, or remove the variable?